src/caesar.js

```diff
@@ -0,0 +1,31 @@
+const caesarCipher = (string, key) => {
+  let decrypt = '';
+  for (let i = 0; i < string.length; i += 1) {
```

Comment:
- [ ] [OPTIONAL] You can also use a `for of` loop for strings. 
[Check this link, ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)it might make your code slimmer.